api_specifications/case-diary-1.0.yaml

```
@@ -0,0 +1,343 @@
+          schema:
+            type: string
+            pattern: "^[a-zA-Z]{2}$"
+            maxLength: 2
+            minLength: 2
+        - name: judgeId
+          in: path
+          description: the Id of the Judge, whose diary is being queried
+          required: true
+          schema:
+            type: string
+            enum: [ "ADiary", "BDiary" ]
```

yamllint (1.35.1)

[error] too many spaces inside brackets

(brackets)